src/loki/status_action.cc

```diff
@@ -44,7 +44,7 @@ void loki_worker_t::status(Api& request) const {
 
   // info that's always returned
   auto* status = request.mutable_status();
-  status->set_version(VALHALLA_VERSION);
+  status->set_version(show_precise_version ? VALHALLA_VERSION_PRECISE : VALHALLA_VERSION);
```

Comment:
do we need to make this configurable? we know if there's a `VALHALLA_VERSION_MODIFIER != ""` so that can be the condition?

Reply:
yep, just updated it. now it's only used when defined, and by default it's not defined unless passed to cmake via the CLI

Reply:
> now I just have to figure out how to leave it unset in CI 

ah yeah, just reading this, that was my worry too. you can just extend the GHA bash scripts with an `CMAKE_EXTRA_ARGS`  or so which you conditionally add to the cmake command?